exercises/03_reverse/reverse.go

```diff
@@ -1,4 +1,19 @@
 package main
 
+import "fmt"
+
+func reverse(n int) int {
+	var s = 0
```

Comment:
Please use `var s int` declaration instead. `0` is the default value; you don't need to assign to `0` explicitly.